platform/android/java/build.gradle

```diff
@@ -0,0 +1,115 @@
+//Gradle project for Godot Engine Android port.
+//Do not modify code between the BEGIN/END sections, as it's autogenerated by add-ons
+
+buildscript {
+	repositories {
+		google()
+		jcenter()
+//CHUNK_BUILDSCRIPT_REPOSITORIES_BEGIN
+//CHUNK_BUILDSCRIPT_REPOSITORIES_END
+	}
+	dependencies {
+		classpath 'com.android.tools.build:gradle:3.2.1'
+//CHUNK_BUILD_DEPENDENCIES_BEGIN
+//CHUNK_BUILD_DEPENDENCIES_END
+	}
+}
+
+apply plugin: 'com.android.application'
+
+allprojects {
+    repositories {
+	mavenCentral()
+	google()
+	jcenter()
+//CHUNK_ALLPROJECTS_REPOSITORIES_BEGIN
+//CHUNK_ALLPROJECTS_REPOSITORIES_END
+
+    }
+}
+
+dependencies {
+	implementation "com.android.support:support-core-utils:28.0.0"
+//CHUNK_DEPENDENCIES_BEGIN
+compile ('com.google.android.gms:play-services-ads:16.0.0') { exclude group: 'com.android.support' }
+//CHUNK_DEPENDENCIES_END
+}
+
+android {
+
+	lintOptions {
+		abortOnError false
+		disable 'MissingTranslation','UnusedResources'
+	}
+
+	compileSdkVersion 28
+	buildToolsVersion "28.0.3"
+	useLibrary 'org.apache.http.legacy'
+
+	packagingOptions {
+		exclude 'META-INF/LICENSE'
+		exclude 'META-INF/NOTICE'
+	}
+	defaultConfig {
+		minSdkVersion 18
+		targetSdkVersion 28
+//CHUNK_ANDROID_DEFAULTCONFIG_BEGIN
+//CHUNK_ANDROID_DEFAULTCONFIG_END
+	}
+	// Both signing and zip-aligning will be done at export time
+	buildTypes.all { buildType ->
+		buildType.zipAlignEnabled false
+		buildType.signingConfig null
+	}
+	sourceSets {
+		main {
+			manifest.srcFile 'AndroidManifest.xml'
+			java.srcDirs = ['src'
+//DIR_SRC_BEGIN
+,'/home/red/coding/godot-demos/2d/platformer/android/admob/src'
+//DIR_SRC_END
+			]
+			res.srcDirs = [
+				'res'
+//DIR_RES_BEGIN
+//DIR_RES_END
+			]
+			aidl.srcDirs = [
+				'aidl'
+//DIR_AIDL_BEGIN
+//DIR_AIDL_END
+			]
+			assets.srcDirs = [
+				'assets'
+//DIR_ASSETS_BEGIN
+//DIR_ASSETS_END
+
+			]
+		}
+		debug.jniLibs.srcDirs = [
+			'libs/debug'
+//DIR_JNI_DEBUG_BEGIN
+//DIR_JNI_DEBUG_END
+		]
+		release.jniLibs.srcDirs = [
+			'libs/release'
+//DIR_JNI_RELEASE_BEGIN
+//DIR_JNI_RELEASE_END
+		]
+	}
+// No longer used, as it's not useful for build source template
```

Comment:
@akien-mga @reduz Since custom builds won't be the default way of building/exporting Godot Android games, is there an alternative to this logic for building Godot Android binary templates and moving them to the `bin` directory?

Also, it's worth noting that this change invalidates the [building export templates](https://docs.godotengine.org/en/latest/development/compiling/compiling_for_android.html#building-the-export-templates) documentation, and so if it's intended to stay, that documentation will need to be updated accordingly.